macros/generate_uuid.sql

```diff
@@ -11,16 +11,16 @@
        SUPPORTS:
             - Postgres
             - Snowflake
-  #}
-  {%- if type == "v4" -%}
+  */#}
+{%- if type == "v4" -%}
     {%- if target.type == 'postgres' -%}
-      (uuid_generate_v4()::text)
+      (uuid_generate_v4()::TEXT)
```

Comment:
Is sqlfluff not telling you uppercase `uuid_generate_v4`? It might be that sqlfluff is considering postgres extension functions similar to user defined ones.